service/controller/v3/resource/instance/template/main.json

```diff
@@ -41,12 +41,6 @@
         "description":"Output value of the master subnet ID as referenced from the virtual network setup."
       }
     },
-    "masterVersionBundleVersions":{
-      "type":"object",
-      "metadata":{
-        "description":"JSON blob of key-value pairs mapping master VMSS instance names to their applied version bundle versions."
-      }
-    },
```

Comment:
Since we rely on the guest API information we do not need to carry these information here anymore.